packages/api/scripts/generate-upload-data.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * This script will generate many uploads for a given user token using the web3.storage API endpoints.
+ * Handy when working with and testing features that require a pristine mass set of data, such as pagination.
+ * Arguments can be passed with a arg=value pattern, ie token='<YOUR TOKEN HERE>'. Passing a token is the only required argument.
+ * Not passing a value will default to a truthy value. ie appending `pin` argument is the same as `pin=true`.
+ * This script requires a users token to be passed or assigned to an environment variable API_TOKEN.
+ * Passable arguments are:
+ * - token: the token of the user these uploads will be authenticated with.
+ * - uploads: The amount of uploads that need to be added, defaulting to 25.
+ * - url: The URL of the API, defaulting to localhost.
+ * - pin: Adds another request for each upload to pin that upload generating pin results, defaulting to false.
+ * Example usage.
+ * From the API folder, run the following command using a generated token.
+ * - node scripts/generate-upload-data.js token='yourApiToken'
+ * This will generate 25 uploads for the given user. Append the pin argument if these uploads should also be pinned.
+ */
+
+import fetch, { Blob } from '@web-std/fetch'
+
+const requiredArgs = [
+  'token'
+]
+
+const optionalArgs = {
+  uploads: '25',
+  pin: false,
+  url: 'http://127.0.0.1:8787'
+}
+
+const argv = process.argv.slice(2).map((arg) => [
```

Comment:
I reckon is ok to keep it as is, but FYI [sade](https://github.com/lukeed/sade) is used across the project to build command line scripts. ie, in scripts/cli.
That would take take of argv parsing, validation, help etc.

Reply:
Ahh sweet, yeah the args here are pretty simple, but if extending it then using sade to parse 'em would be amazing. Thanks!